src/components/Editor/GutterMenu.js

```diff
@@ -154,6 +153,19 @@ class GutterContextMenuComponent extends PureComponent {
   }
 }
 
+const {
```

Comment:
this would be better done in line 19

Reply:
@codehag 
Is it required to move this up?
Put here for the following reasons:
- closer to where the actions are used, which is `connect` call
- put at the top, the eslint would complain the no-shadow issue. (this has also been mentioned in the issue description [1])
- this style might be thought 
  ```js
  import { ... } from actions
  ```
   but we will face some transpile issue [2]

[1] https://github.com/devtools-html/debugger.html/issues/4532#issue-269498439

[2] https://github.com/devtools-html/debugger.html/pull/4640#discussion_r149972541